content/doc/addons/materia-db-kv.md

```diff
@@ -163,3 +163,6 @@ Find below the list of currently supported commands:
 | `STRLEN` | Returns the length of the string value stored at `key`. An error is returned when key holds a non-string value. |
 | `TTL` | Returns the remaining time to live of a key that has a timeout. During the alpha phase, key timeouts aren't implemented. Nevertheless, for compatibility reasons, we support the `TTL` command. The command returns -2 if the key does not exist, it returns -1 if the key exists with no defined timeout. |
 | `TYPE` | Returns the string representation of the type of the value stored at `key`. Can be: `hash`, `list` or `string`.
+
+ {{% content/managed-services %}}
```

Comment:
As it's a shared add-on, it maybe more complicated to provide "custom" configuration for Materia KV add-ons. cc @PierreZ should we add a more custom note for it?

Reply:
Minimal customization can be done on add-ons that are spawned in a multi-tenant way. I think we should split the shortcode in two:

* `{{% content/dedicated-managed-services %}}` for current dbaas-oriented products and others
* `{{% content/materia-services %}}` for current materia-oriented product

Reply:
Deal, let's remove the shortcode on Materia pages for now.

Reply:
Thanks!